README.md

```diff
@@ -54,6 +51,12 @@ This will:
 
 Then the tests must be compiled with `CFG_GP_PACKAGE_PATH=<path>`.
 
+It makes use of the following environment variable:
+* `COMPILE_NS_USER`: `32` or `64` if application shall be compiled in AArch32 on in
+  AArch64 mode. If not specificed, build relies on `CFG_ARM32_core=y` from
+  OP-TEE core build to assume application is AArch32. Otherwise, AArch64 is
+  assumed.
```

Comment:
Minor: using Aarch32 and Aarch64 in this context is incorrect, as they are execution states of ARMv8. It would be better to write "32 bits" and "64 bits" which is a bit imprecise but good enough for the purpose.